src/test/obj_pmalloc_mt/obj_pmalloc_mt.c

```diff
@@ -155,6 +155,39 @@ alloc_free_worker(void *arg)
 	return NULL;
 }
 
+#define OPS_PER_TX 10
+#define TX_PER_TH 100
+#define STEP 8
+#define TEST_LANES 4
+
+static void *
+tx2_worker(void *arg)
+{
+	struct worker_args *a = arg;
+
+	for (int i = 0; i < TX_PER_TH; ++i) {
+		PMEMoid oids[OPS_PER_TX];
+		TX_BEGIN(a->pop) {
+			for (int i = 0; i < OPS_PER_TX; ++i) {
+				oids[i] = pmemobj_tx_alloc(ALLOC_SIZE, a->idx);
+				for (int j = 0; j < ALLOC_SIZE; j += STEP) {
+					pmemobj_tx_add_range(oids[i], j, STEP);
```

Comment:
As I see, `ALLOC_SIZE` is 100, and `STEP` is 8, which is not a divisor of 100. Won't this be a problem, at least in principle?
The last iteraton is:
`pmemobj_tx_add_range(oids[i], 96, 8);` <- adding 4 bytes over the allocation.